fossunited/ticketing/doctype/foss_event_ticket/foss_event_ticket.json

```diff
@@ -157,6 +176,12 @@
    "role": "System Manager",
    "share": 1,
    "write": 1
+  },
+  {
+   "export": 1,
+   "read": 1,
+   "role": "Chapter Lead",
+   "select": 1
```

Comment:
Is this necessary to ensure that check-ins can be setup? Not sure why this new permission is needed

Reply:
Yeah, this is unnecessary now that i see it. Removing this permission